doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+MAXHEIGHT(600) | E | 600 | Maximum pixel height for GUI dialog windows
+MAXLCL(100000) | M | 100000 | Maximum local macro variables
+MAXLINE(200000) | MA | 200000 | Maximum MLC, MAC, CPY, and BAL source lines that can be loaded into memory during an assembly.
+MAXLOG(1000000) | MALE | 1000000 | Maximum GUI log file output before truncation begins by removing 50% to limit memory consumption. The only limit on size of log file is MAXSIZE.
+MAXPARM(100000) | M | 10000 | Maximum length of any string during macro processing.  This is an extension as HLASM limit is currently 1024.
+MAXPASS(2) | A | 2 | Maximum passes of the BAL source by assembler to resolve forward nested symbol references prior to final pass to generate object code.  The number of LOCTR statements is added to this limit in order to resolve the final address of each LOCTR section.
+MAXPC(50000) | M | 50000 | Maximum pseudo code instructions held in cache before LRU replacement begins.
+MAXQUE(1000) | E | 1000 | Maximum output queue length from any CMD started task before queue messages are automatically copied to GUI output log to conserve memory and prevent stall.
+MAXRLD(10000) | L | 10000 | Maximum RLD relocation records allowed in a program.
+MAXSIZE(50)| MALE | 50 | Maximum size of any output file in MB
+MAXSYM(50000) | MA | 50000 | Maximum symbols in one macro assembly
+MAXWARN(4) | MA | 4 | Maximum MNOTE warning level without generating error.
+MAXWIDTH(800) | E | 800 | Maximum pixel width for GUI dialog windows
+MCALL | A | NO | List each macro call and exit on the PRN assembly listing in addition to first level calls if LISTCALL option is also on.
+MEM(1) | E | 1 | Memory allocated for 390 program execution in MB.  Note for RMODE31 loads and GETMAIN's memory above the 16 MB line must be allocated.  For example MEM(32) would provide 16 MB below the line and 16 MB above.
+MINHEIGHT(150) | E | 150 | Minimum pixel height for GUI dialog windows
+MINWIDTH(150) | E | 150 | Minimum pixel width for GUI dialog windows
+MNOTE(0) | MA | 0 | Control MNOTE error/warning messages during macro expansion and assembly: <br/> 0 - default generates MNOTE error/warning during mz390 and az390 <br/> 1 - suppress mz390 error/warning and only pass MNOTE to az390 <br/> 2 - generates MNOTE error/warning during mz390 and suppresses passing them on to az390
+MOD | L | NO | Generate raw code file from lz390 with .MOD suffix and no header or trailer and no RLD's. 
+OBJ | A | YES | Generate relocatable object code file
+OBJHEX | A | NO | Generate ASCII readable form of OBJ file with support for 31 bit long sections.  The default is to generate binary OBJ file which is compatible with mainframe linkers but is limited to 24 bit long sections.
+PARM('text') | E | none | Define character string parm to passed to executable program via address in R1 at startup pointing to a fullword pointing to a halfword length followed by EBCDIC characters. When the length is zero, no PARM is present.
```

Comment:
Line 74. First sentence. "... parm to passed to ..." -> "... parm passed to ...".

Reply:
Fixed 40c73aa